### ISSUE TITLE
Route order not preserved when logging ascents

### ISSUE THREAD

**Issue by killakalle** [theCrag/website#3915]
**What happened?**
TL;DR
Route order is not preserved when logging ascents

I climbed two 6a's next to each other. One I managed in one go, for the other one I took two attempts. For the two routes I logged all three ascents. When going through the ascent comments of one of the routes, I realized that my comments were describing the "wrong" routes. I then realized, that I had mixed up the two routes. However, when I had a closer look, I realized that actually theCrag had the order of routes reversed when logging ascents. Both were 6a's and I did not pay particular attention to the names as the routes are nothing special.

Example url(s) to reproduce the problem:
https://www.thecrag.com/es/escalar/spain/valencia/gestalgar/area/2561079678

Ideally also provide screenshots:

Order on route list
![image](https://user-images.githubusercontent.com/30682796/118413765-37609b00-b6a1-11eb-8b56-96b97179fc9c.png)

When logging ascents the order of routes is reversed
![image](https://user-images.githubusercontent.com/30682796/118413781-4e9f8880-b6a1-11eb-88e1-e276272ee096.png)

**What you expected:**

* [ ] When logging ascents, the order of routes is the same as the order on the route list

**Comment by skomaroff** [comment]
I ran into the same issue at a different crag yesterday.

**Comment by quaestor** [comment]
+1. This is really annoying, and I believe this is a regression.

**Comment by georg-d** [comment]
+1 because it's quite demanding not to mix up anything - especially on a mobile with too limited display space to show route listing and ascent loggin side by side - when you log all 12 ascents of a day at once. That happens usually when there's no mobile network coverage at a crag, so you can't log ascents directly after you did them, but you've to log all ascents after leaving the crag.

**Comment by killakalle** [comment]
This happens also when bulk editing routes.

![image](https://user-images.githubusercontent.com/30682796/138277430-ddc9bc64-cd4b-4a02-82d4-093c6f256d08.png)

![image](https://user-images.githubusercontent.com/30682796/138277623-89c03554-4cda-4c36-9f69-4d2b78657a5d.png)

**Comment by quaestor** [comment]
Bumping this. This used to work correctly, please prioritize. I know you're working on the new tick interface, but most users are still using the old one (including me), and as @killakalle reported, this also affects bulk editing, which is not going away in the near future, I guess?

**Comment by quaestor** [comment]
Another bump. I'm annoyed by this each time I log something.

**Comment by rouletout** [comment]
We are aware of that but this will not be fixed anymore as the new ticking interface fixes it and will be released rather sooner than later.

**Comment by quaestor** [comment]
@rouletout As I said above and @killakalle pointed out, too, it also affects the order when bulk editing, which isn't going to go away any time soon? I think it's essential to fix this.

**Comment by scd** [comment]
Because this is causing distress and it looked like it was mixed up with bulk edit I looked into this.

I have fixed the logging ascent order so unless there is another broken path it should be good now. I did 10 tests on dev and did not find an out of order sequence. Please test yourselves though.

If there is a problem with bulk edit it is completely unrelated to the fix. I could not replicate the problem nor could I find some obvious broken code (as per ascent logging). 

I am therefore now closing this issue but if you find the problem is still occurring in bulk edit then please create another card, otherwise the issue will be lost in the noise like before. Only open this card again if there is still an ascent logging order issue.

**Comment by quaestor** [comment]
Thanks Simon for fixing this! I did a few tests and can confirm that everything is in order now (pun intended).

I can confirm that there seems to be no issue with bulk edit (any more?).
